flocker/node/agents/blockdevice.py

```diff
@@ -1198,13 +1266,14 @@ def create_volume(self, dataset_id, size):
         See ``IBlockDeviceAPI.create_volume`` for parameter and return type
         documentation.
         """
+        actual_size = allocated_size(self.allocation_unit(), size)
```

Comment:
I don't think you should be doing rounding in loopback implementation. Instead, you should throw exception if requested size is not multiple of allocation unit.

At least some backends *cannot* do rounding, e.g. ScaleIO. We therefore want to ensure that we never call create_volume() with something that isn't multiple of `allocation_unit()`. So I think loopback should implement the more restrictive behaviour so it can catch bugs in the deployer.